lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4957,6 +4988,7 @@ SwiftDeclConverter::importSwiftNewtype(const clang::TypedefNameDecl *decl,
   auto structDecl = Impl.createDeclWithClangNode<StructDecl>(
       decl, AccessLevel::Public, Loc, name, Loc, None, nullptr, dc);
   structDecl->computeType();
+  structDecl->setValidationStarted();
```

Comment:
For cases that do go through `createDeclWithClangNode`, can we set this there instead? Or are there callers that still rely on Sema-based validation?

Reply:
That's a good idea. I would hope we don't rely on Sema validation here, since it would be break if there's no Sema. I'll give it a shot and see if things work